### ISSUE TITLE
Config is not writable on production

### ISSUE THREAD

**Issue by jhhazelaar** [lwekuiper/statamic-activecampaign#3]
The addon saves the Form settings in the config `statamic.activecampaign.forms`, but on webservers the application can't change php files and it's a security risk to enable this.

With statamic git automation the config file isn't tracked and with every deployment (If you use Envoyer or deployer) you loose your form settings.

Why are you saving this in de config?

**Comment by lwekuiper** [comment]
The [edalzell/statamic-forma](https://github.com/edalzell/statamic-forma) package served as inspiration to save the form settings in config. The mailchimp addon used to do the same, until two weeks ago. Saving the settings in yaml files might solve this, what do you think?

**Comment by jhhazelaar** [comment]
Yes that's solved the problem. When saved in a yaml file in content the git automation also picks it up.

Other think, maybe other issue. We have 50 forms that needs to be connected and one form to load the config is really slow. So if you save it in yaml file, please don't do it in one file. But just as https://github.com/riasvdv/statamic-redirect every config of a form in een separate file.

**Comment by lwekuiper** [comment]
Thanks for the feedback. I'll definitely have a look at refactoring this to yaml files.

**Comment by Klaasc7** [comment]
Do you have any updates on the status of this bug?

**Comment by lwekuiper** [comment]
I expect to have some time this week to work on this.

**Comment by Klaasc7** [comment]
Do you have any updates on the status of this bug?

**Comment by lwekuiper** [comment]
Working on version 2 which uses a custom stache to store the ActiveCampaign configurations, similar to the redirect addon as mentioned by @jhhazelaar. It's looking really good, but will still take some more work to finish.

**Comment by lwekuiper** [comment]
Just released v2.0.0-beta.1 which saves the configurations in individual yaml files.

**Comment by lwekuiper** [comment]
This issue has been resolved in release v2.0.0.
